<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontAxisPanel</class>
 <widget class="QWidget" name="FrontAxisPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableViewAxis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1241</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>471</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widgetFilter" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="widgetTransfer" native="true"/>
     </item>
    </layout>
   </widget>
   <zorder>widgetFilter</zorder>
   <zorder></zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
